<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>691</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1008</width>
    <height>691</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1008</width>
    <height>691</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RegisterWindow
{
background-color:rgb(0, 157, 255)
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>25</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label
{
color:white;
}</string>
     </property>
     <property name="text">
      <string>  Welcome to SDMS</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>306</width>
          <height>600</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#groupBox
{
	font: 15pt &quot;Segoe UI&quot;;
background:white;
}</string>
        </property>
        <property name="title">
         <string>Registration</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Please select your role</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QRadioButton" name="radioButton">
                  <property name="text">
                   <string>Admin</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton_2">
                  <property name="text">
                   <string>Student</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameLine">
             <property name="toolTipDuration">
              <number>-1</number>
             </property>
             <property name="styleSheet">
              <string notr="true">#nameLine
{
border-radius:4px;
border:1px solid black;
padding:3px
}</string>
             </property>
             <property name="cursorPosition">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Surname</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="surnameLine">
             <property name="styleSheet">
              <string notr="true">#surnameLine
{
border-radius:4px;
border:1px solid black;
padding:3px
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Date of birth</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit">
             <property name="styleSheet">
              <string notr="true">#dateEdit
{
border-radius:4px;
border:1px solid black;
padding:3px
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Email Address</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="emailLine">
             <property name="styleSheet">
              <string notr="true">#emailLine
{
border-radius:4px;
border:1px solid black;
padding:3px
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwordLine">
             <property name="styleSheet">
              <string notr="true">#passwordLine
{
border-radius:4px;
border:1px solid black;
padding:3px;
text-type:password;
}</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Confirm Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="conPasswordLine">
             <property name="contextMenuPolicy">
              <enum>Qt::DefaultContextMenu</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">#conPasswordLine
{
border-radius:4px;
border:1px solid black;
padding:3px
}</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Already Registered?</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="signInButton">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">#signInButton
{
background-color:white;
color:green;
border-radius:1px

}</string>
             </property>
             <property name="text">
              <string>Sign In</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="cancelButton">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">#cancelButton
{
	background-color:rgb(0, 157, 255);
color:white;
border:1px solid rgb(0, 157, 255);
width:65px;
padding:5px;
}
#cancelButton:hover
{
border:rgb(0, 87, 145);
background-color:rgb(0, 87, 145);
color:white;
transition: all .5s ease;

}</string>
             </property>
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nextButton">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">#nextButton
{
	background-color:rgb(0, 157, 255);
color:white;
border:1px solid rgb(0, 157, 255);
width:65px;
padding:5px;
}
#nextButton:hover
{
border:rgb(0, 87, 145);
background-color:rgb(0, 87, 145);
color:white;

}</string>
             </property>
             <property name="text">
              <string>Next</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
